cmd/run_test.go

```diff
@@ -125,3 +134,85 @@ func TestHandleSummaryResultError(t *testing.T) {
 	assertEqual(t, "file summary 1", files[filePath1])
 	assertEqual(t, "file summary 2", files[filePath2])
 }
+
+func TestAbortTest(t *testing.T) { //nolint: tparallel
+	t.Parallel()
+
+	testCases := []struct {
+		testFilename, expLogOutput string
+	}{
+		{
+			testFilename: "abort.js",
+		},
+		{
+			testFilename: "abort_initerr.js",
+		},
+		{
+			testFilename: "abort_initvu.js",
+		},
+		{
+			testFilename: "abort_teardown.js",
+			expLogOutput: "Calling teardown function after test.abort()",
+		},
```

Comment:
nitpick: for these really small testfiles I kind of feel like moving them to files just adds extra steps for whoever looks at the code. AFAIK the reason for this to be the case for other tests is that the testdata is quite ... big and hard to put in a string in the code, none of this is true for this couple of lines here.

Reply:
I agree, but these tests test the `runCmd`, so I can't see how we could turn the JS into strings... Writing to `os.Stdin` maybe? :confused: